Compatibility/MacBookAir5,1.json

```
@@ -0,0 +1,20 @@
+{
+    "MacName": "MacBook Air (11-inch, Mid-2012)",
+    "Author": "@Blueysh",
+    "Approved": [],
+    "Details": "Performance is great. WiFi and Bluetooth will not work out of the box, but Patched Sur can fix by running Patch Kexts in the post-install app. Battery life isn't perfect but that's expected for older machines.",
+    "Works": [
+        "WiFi and Bluetooth",
+        "I/O Ports",
+        "Graphics Acceleration",
+        "Sleep and Brightness"
+    ],
+    "Unknowns": [
+        "Ethernet (not tested)"
+    ],
+    "Warns": [
+        "FileVault",
+        "Personal Hotspots (There is a workaround)",
```

```suggestion
        "Personal Hotspots",
```
There isn't room for that part, otherwise it's good.